style.css

```
@@ -0,0 +1,119 @@
+* {
+    box-sizing: border-box;
+    font-family: 'IBM Plex Sans', sans-serif;
+}
+/* 
+html,
+body{
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+} */
+
+body {
+    margin: 0;
+    background: rgb(116,255,0);
+    background: linear-gradient(0deg, rgba(116,255,0,1) 0%, rgba(255,255,255,1) 100%) fixed;
+}
+
+h1 {
+    font-size: 25px;
+    text-transform: uppercase;
+    padding-top: 3px;
+    letter-spacing: 1px;
+}
+
+h2 {
+    font-size: 20px;
+    padding-bottom: 6px;
+    text-transform: uppercase;
+    letter-spacing: 3px;
+}
+
+h3 {
+    font-size: 16px;
+    margin-bottom: 0;
+}
+
+h4 {
+    color: #333333;
+    font-style: italic;
+    font-weight: 400;
+    font-size: 15px;
+    margin-bottom: 10px;
+    margin-top: 5px;
+}
+
+p,
+li {
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 1.7;
+}
+
+p {
+    padding-left: 30px; 
+    margin-top: 7px;
+}
+
+ul {
+    list-style: none;
+    padding-left: 35px;
+    margin-top: 7px;
+
+}
+
+a {
+    color: black;
+    text-decoration: none;
+}
+
+.wrapper {
+    /* height: 100%; */
+    max-width: 1200px;
+    /* max-height: 800px; */
+    /* border: 1px solid red; */
+    margin: auto;
+    position: relative;
+}
+
+/* info */
+#info {
+    /* background-color: #6480E3; */
+    border-right: 1px solid black;
+    width: 200px;
+    height: 100%;
+    padding-left: 25px;
+    position: fixed;
+}
+
+#info ul{
+    padding-left: 0;
+}
+
+/* content */
+#cotent-wrap {
+    width: 100%;
+    /* height: 100%; */
+    /* max-height: 800px; */
+    padding-left: 200px;
+    /* overflow: auto; */
+    /* position: relative; */
+}
+
+.column {
+    width: 50%;
+    float: left;
+}
```

I usually reserve floating for elements that need to have text wrapped around them; here, instead, the effect you want is to have side-by-side elements and `display:inline-block` will get you that 👍🏻

```suggestion
.column {
    display: inline-block;
    vertical-align: top;
    width: 49.5%;
}
```

Setting the `vertical-align` will align all your columns to the top vs the text baseline, and using `inline-block` also means you have to tweak the width slightly.

But this avoids the layout hassles that come with `float` _and_ means you don't need to use [clearfix](https://mica-web.github.io/learn/css/best-practices#wrap-all-floated-elements-in-another-element-with-the-clearfix-class-applied).